workspaces/services/guacamole-azure-dsvm/terraform/main.tf

```diff
@@ -0,0 +1,33 @@
+# Azure Provider source and version being used
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.64.0"
+    }
+  }
+  backend "azurerm" {
+  }
+}
+
+
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_client_config" "current" {}
+
+data "azurerm_resource_group" "ws" {
+  name = "rg-${var.tre_id}-ws-${var.workspace_id}"
+}
+
+data "azurerm_virtual_network" "ws" {
+  name                = "vnet-${var.tre_id}-ws-${var.workspace_id}"
+  resource_group_name = data.azurerm_resource_group.ws.name
+}
+
+data "azurerm_subnet" "dsvms" {
```

Comment:
Is there a reason for a new subnet? Can we just put them on the `ServicesSubnet` please? Otherwise each time someone adds a service VNET configurations will need changing and won't be possible to add the service to existing vnets.